.github/workflows/deploy.yml

```diff
@@ -0,0 +1,69 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: [Docker build and publish]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: "Configure"
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: ${{vars.AWS_REGION}}
+          aws-access-key-id: ${{ secrets.DEPLOY_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.DEPLOY_ACCESS_KEY }}
+
+      - name: install oathtool
+        run: sudo apt-get install -y oathtool jq
+
+      - id: auth
+        name: Authenticate to STS
+        run: |
+          mfa_serial="arn:aws:iam::${{vars.AWS_ACCOUNT_NUMBER}}:mfa/gh-action"
```

Comment:
`gh-action` should be an environment variable.

I see mfa/gh-action is tied to your AWS username. From our Slack conversation, we need another GH Ticket to track creating an AWS user that is isolated for this GH Action CI pipeline, and that would have its own mfa/<name>, which would be great if its editable as an env var.

Reply:
done